src/main/java/use_cases/PlayerGameInteractor.java

```diff
@@ -110,9 +117,9 @@ private void playCardRequestLogic(Card chosenCard, Player currentPlayer) {
      * can pick up a Card, and if they can, then pick one up. Otherwise, the function won't do anything.
      */
     private void pickUpCardRequestLogic(Player currentPlayer) {
-        if(!(currentGame.getCurrentTurnHasPickedUp()) && !(anyValidCards(currentPlayer))) {
-            currentPlayer.pickUpCard(currentGame);
-            currentGame.notifyGameObservers();
+        if(!(access.getCurrentTurnHasPickedUp()) && !(anyValidCards(currentPlayer))) {
+            access.pickUpCard(currentPlayer);
+            notifier.update();
```

Comment:
I'm wondering what would be the difference between calling access.notifyGameObservers() and notifier.update() ?

Reply:
the purpose of the notifier class is purely to hide details of an observable object from the objects that _only_ want to notify its observers. essentially, this interactor shouldnt care about adding/removing observers to this game, it should only tell the observers theres been an update

the GameAccess interface it has a reference to also doesnt include an update observers method, thus it needs a notifier